### ISSUE TITLE
[Feature request] One topic assigned to multiple modules

### ISSUE THREAD

**Issue by dmusdev** [tecnickcom/tcexam#328]
Hello, we would like to use modules for classes (module 1 for class of 30 students, module 2 for another class of different 30 students, etc.).

In teaching English, many topics are often revised (repeated) in multiple years. Therefore, we would like a topic to be possible to assign to multiple modules.

This does not seem to be possible, if we haven't missed something.

Thank you!

**Comment by nicolaasuni** [comment]
When creating a test you can combine any topic from any module.
The test editor allows you to cherry pick and combine any topic from any module to for a custom test.

**Comment by dmusdev** [comment]
I know that you can create custom tests. But that's not what I requested.

Let me rephrase: I want **one single** topic to be possible to be **permanent** member of _multiple different modules_.
